core/tests/test_utils.py

```diff
@@ -305,6 +307,14 @@ def get_current_logged_in_user_id(self):
     def get_user_id_from_email(self, email):
         return current_user_services.get_user_id_from_email(email)
 
+    def get_gravatar_from_url(self, url):
```

Comment:
Method to fetch gravatar from a url provide (Used in test_signup_gravatar method in profile_test.py)